kimchi/src/prover.rs

```diff
@@ -73,13 +80,23 @@ where
         mut witness: [Vec<ScalarField<G>>; COLUMNS],
         index: &ProverIndex<G>,
         prev_challenges: Vec<(Vec<ScalarField<G>>, PolyComm<G>)>,
+        blinders: [Option<PolyComm<ScalarField<G>>>; COLUMNS],
```

Comment:
why are you doing this?

Reply:
If you want we could make it a separate method with a scary name? It would be nice to merge it in though

Reply:
If you want we could make it a separate method with a scary name? My preference would be to merge it but it's OK if we don't

Reply:
I added a wrapping `Option<[Option<.>; COLUMNS]>` for now